modules/SFML/Config.cppm

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file Config.cppm
+ * @brief Module file for SFML macro constants.
+ */
+
+module;
+
+#include <SFML/Config.hpp>
+
+export module sfml.Config;
```

Comment:
Is lower case a C++ recommendation or why not `SFML.Config` / `SFML.Audio` / etc. ?